### ISSUE TITLE
Support [[link]]s shorthand

### ISSUE THREAD

**Issue by grygoriy-pinkas-scx** [spencermountain/wtf_wikipedia#316]
Hello! Script returns the wrong text of link for the article "Dutch language"!
![Screen Shot 2019-10-11 at 14 31 31](https://user-images.githubusercontent.com/48279096/66648648-bd4e7180-ec34-11e9-90cc-1a3bcb2a9cab.png)
![Screen Shot 2019-10-11 at 14 32 35](https://user-images.githubusercontent.com/48279096/66648658-c5a6ac80-ec34-11e9-98d9-58dba270a6e2.png)

**Comment by spencermountain** [comment]
hey @grygoriy-pinkas-scx thanks - yeah this is a shorthand in wikipedia:
```
let str = `majority of [[music]], [[film]]s, [[book]]s`
let doc = wtf(str)
console.log(doc.links())
```
if i can remember, i think it is supported in html output, but maybe we should co-erce the trailing s into the link json too. I'm not sure.

fwiw, i think `[[link]]'s` is a similar situation.
cheers

**Comment by Istador** [comment]
When I remember it correctly that is implemented by an wiki extension and not all wikis might have it. If this issue is implemented, the parser should have an option for turning it off in case you want to parse a wiki that doesn't support this syntax.

**Comment by spencermountain** [comment]
hey @Istador i think this is fixed now in 8.1.0
```js
let str = `majority of [[music]], [[film]]s, [[book]]s`
let doc = wtf(str)
console.log(doc.links())
/*[ { text: 'music', type: 'internal', page: 'music' },
  { text: 'films', type: 'internal', page: 'film' },
  { text: 'books', type: 'internal', page: 'book' } ]
*/
```
thanks
